### ISSUE TITLE
User is not notified of failed register

### ISSUE THREAD

**Issue by asp78** [mentii/mentii#67]
If you register a user, activate, and try to register a user with the same email, the user is not presented with any information as to why the register button seemingly stopped working

**Comment by jonathanmcdaniel** [comment]
This will be a part of Frontend Notifications #48. At the time of writing the registration the backend wasnt returing anything useful to determine what went wrong so it just kinda falls off. May also be somewhat resolved with Create a User Service to abstract away user related API calls (3) #55.

**Comment by asp78** [comment]
I'm not sure if this should wait until #48, as it should have been in with the release of the story. It would just be red text similar to 'Passwords don't match' to notify that register failed. That Notifications story is more of an overhaul and placing a framework in. It may be easier to complete after #55 but shouldn't be implemented as part of that story.

**Comment by asp78** [comment]
This is a duplicate of #63, closing.
